### PR TITLE
K8s: delete RERC

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-2620

### CODE REVIEW

File: content/operate/kubernetes/active-active/delete-rerc.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+weight: 98
+---
+  
+When a RedisEnterpriseRemoteCluster (RERC) isn't communicating with other participating clusters, the RedisEnterpriseActiveActiveDatabase(REAADB) will enter an error state. To remedy this, you can remove the unresponsive RERC from the Active-Active database configuration.
+
+## Prerequisites
+
+We strongly recommend backing up your cluster before proceeding with the following steps.
+
+{{<warning>}}Proceed with caution. Ensure you are running the following command from the correct instance. Verify the status of the RERCs and REAADB before and after each step.{{</warning>}}
```
Comment (andy-stark-redis) [inline content/operate/kubernetes/active-active/delete-rerc.md:19]:
```suggestion
{{<warning>}}Proceed with caution. Ensure you are running the following commands from the correct instance. Verify the status of the RERCs and REAADB before and after each step.{{</warning>}}
```

File: content/operate/kubernetes/active-active/delete-rerc.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+{{<warning>}}Use with caution and verify you are purging the correct instance.{{</warning>}}
+
+```sh
+crdb-cli crdb purge-instance --crdb-guid  <REAADB_GUID> --instance-id <INSTANCE_ID>
+```
+
+## Next steps
+
+After the communication issue on the cluster is resolved, you can add it back to the REAADB configuration by following the steps for [Add a participating cluster]({{<relref "operate/kubernetes/active-active/edit-clusters#add-a-participating-cluster">}}.
```
Comment (andy-stark-redis) [inline content/operate/kubernetes/active-active/delete-rerc.md:128]: Missing ")" at end of link:
```suggestion
After the communication issue on the cluster is resolved, you can add it back to the REAADB configuration by following the steps for [Add a participating cluster]({{<relref "operate/kubernetes/active-active/edit-clusters#add-a-participating-cluster">}}).
```

**Review by andy-stark-redis — APPROVED** [review]
Minor glitch, but otherwise looks good :-)

**Review by andy-stark-redis — APPROVED** [review]
LGTM

**Comment by kaitlynmichael** [comment]
Closing for now until dev and support decide on a course of action.

**Pull request closed without merging**